EISGIT error when trying to deploy to Vercel: Appears to be a git repo or submodule.

##### Environment
* Next.js
* Dash.js version: 4.0.0-npm

##### Steps to reproduce
1. Install the latest version of dash.js using npm
2. Try to deploy to Vercel

##### Observed behavior
Build fails throwing the `EISGIT` error.

Regenerating `package-lock.json` or ignoring build cache resolves the issue for the initial deployment, the second one will fail again.

##### Console output
```
npm ERR! code EISGIT
npm ERR! path /vercel/path0/node_modules/dashjs
npm ERR! git /vercel/path0/node_modules/dashjs: Appears to be a git repo or submodule.
npm ERR! git     /vercel/path0/node_modules/dashjs
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

No problems observed on local machine.

--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
         "test": "mocha test/unit --require mochahook",
         "test-browserunit": "karma start build/karma.conf.js",
         "test-functional": "node test/functional/runTests.js --selenium=remote --app=remote",
-        "postinstall": "node githook.js",
+        "prepare": "node githook.js",
         "prepack": "npm run build"
     },
     "devDependencies": {

